ISSUE_DRAFT.md

```diff
@@ -0,0 +1,131 @@
+# Add Linux ARM64/aarch64 Architecture Support
```

Comment:
this file looks like it doesn't belong to the PR, can you remove it?

Reply:
You're absolutely right - ISSUE_DRAFT.md was accidentally included and doesn't belong in this PR. Removed it from the commit.